packages/ui-library/src/components/sidebar-navigation/mobile.js

```diff
@@ -55,6 +60,7 @@ Mobile.propTypes = {
 	children: PropTypes.node.isRequired,
 	openButtonScreenReaderText: PropTypes.string,
 	closeButtonScreenReaderText: PropTypes.string,
+	"aria-label": PropTypes.string,
```

Comment:
I'm for all these being _required_ props. This will ensure people not forgetting about adding translated versions of them when using the mobile component.
I think the fact that you got confused with the Settings here only speaks for changing this.
However, this does change the API, and thus increases the scope. What do you think? Should we create a new issue?